src/test/java/address/testutil/ContextMenuChoice.java

```diff
@@ -0,0 +1,8 @@
+package address.testutil;
+
+/**
+ *
+ */
+public enum ContextMenuChoice {
+        EDIT, TAG, DELETE, CANCEL;
+}
```

Comment:
Maybe i will remove this. Will have four different methods instead of one unified method